source/install/desktop-app-changelog.rst

```diff
@@ -3,12 +3,89 @@ Desktop Application Changelog
+
+Breaking Changes / Upgrade Notes
+~~~~~~~~~~~~~~~
+
+- Some keyboard shortcuts and menu items were updated to work with the new Desktop App layout. ``Ctrl+#`` is used for changing tabs and ``Ctrl+Shft+#`` is used for changing servers.
+
+Highlights
+~~~~~~~~~~~~~~~
+
+- Redesigned title bar allows users to seamlessly work in Channels, Playbooks, and Boards across multiple servers with minimal context switching.
+
+Improvements
+~~~~~~~~~~~~~~~
+
+MacOS
+^^^^^^
+- Made the window menu on macOS more consistent with system standards.
+
+All Platforms
+^^^^^^^^^^^^^
+- Added support for multiple languages to be used by the spellchecker. This can be configured in the desktop preferences.
+- Updated loading screen visuals.
+- Added a dark mode for settings and modals.
+- Changed the server selection to use a dropdown instead of tabs.
+- Added support for dragging and dropping of the server dropdown items to re-order servers.
+- Converted the tabs interface to support multiple configurable tabs based on the added server to easily access Boards and Playbooks via tabs in the window header.
+- Removed the **Server Management** screen from **Settings**, and added Edit/Delete buttons to the new dropdown, as users can now configure and edit their servers from the server dropdown menu.
+- Added a checkbox to certificate error modal that allows users to permanently distrust a certificate.
+
+Architectural Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Major version upgrade of Electron to v14.1. Electron is the underlying technology used to build the Desktop app.
+- Added a RPM build option to the Electron builder.
+- Added Universal binaries for MacOS users.
+- Migrated to Bootstrap v4 and refreshed the interface. Migrated to ``react-beautiful-dnd`` instead of ``react-smooth-dnd`` for a cleaner experience.
+
+Bug Fixes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Linux
+^^^^^^^^^^^^^
+- Fixed the tray icon size on Linux.
+- Fixed an issue where pressing ``Alt+<somekey>`` could cause the menu bar to disable and overlap the top bar on Linux.
+
+All Platforms
+^^^^^^^^^^^^^
+- Fixed an issue where resizing the app while in the System Console caused a white bar to appear at the top.
+- Fixed an issue where the right-click menu was missing from the ``jira connect`` modal.
+- Fixed an issue where the app would render off screen and the user would have trouble getting the window in view.
+
+Known Issues
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- The new spellchecker connects to Google servers for downloading updated dictionaries.
+- Unread messages icon may be missing from the taskbar on Windows following 4.7.0 upgrade `MM-37807 <https://mattermost.atlassian.net/browse/MM-37807>`_.
+- The ``create_desktop_file.sh`` script is removed from the .tar.gz release. As a workaround, it can be downloaded from `GitHub here <https://github.com/mattermost/desktop/blob/master/src/assets/linux/create_desktop_file.sh>`_.
```

Comment:
@devinbinnie didn't we fix this one? is it still missing? it should be a one line change or something like that.